data_steward/utils/sandbox.py

```diff
@@ -42,7 +42,10 @@ def get_sandbox_table_name(table_namer, base_name):
     :param base_name: the name of the cleaning rule
     :return: the concatenated table name
     """
-    return f'{table_namer}_{base_name}'
+    if table_namer is None or table_namer == '':
+        return base_name
+    else:
+        return f'{table_namer}_{base_name}'
```

Comment:
Feel like we can use python truthy-ness and falsy-ness for this comparison.  I don't recommend changing the function name or signature.
```
def get_name(table_namer, base_name):
    if table_namer is None or table_namer == '':
        return base_name
    else:
        return f'{table_namer}_{base_name}'
 
get_name(None, 'base')        #  returns "base"
get_name('', 'base')          #  returns "base"
get_name(' ', 'base')         #  returns " _base"


# using python truthy-ness                                                                                                                                                                                  
def get_new_name(table_namer, base_name):
  if table_namer and not table_namer.isspace():
    return f'{table_namer}_{base_name}'
  
  return base_name
 
get_new_name(None, 'base')        #  returns "base"
get_new_name('', 'base')          #  returns "base"
get_new_name('   ', 'base')       #  returns "base"
```

Reply:
Nice catch, I'll change this.

Reply:
How does it look now?